tests/integration/telemetry/stats/prometheus/nullvm/istioctl_metrics_test.go

```diff
@@ -56,7 +56,7 @@ func validateDefaultOutput(t framework.TestContext, workload string) error { //
 	}
 
 	// output will be something like
-	//      WORKLOAD    TOTAL RPS    ERROR RPS  P50 LATENCY  P90 LATENCY  P99 LATENCY
+	//          NAME    TOTAL RPS    ERROR RPS  P50 LATENCY  P90 LATENCY  P99 LATENCY
```

Comment:
do we need an new column like `resource type` or something?

Reply:
I reference `kubectl get` cmd，only input/output  `Name` will be more intuitive.

```bash
kubectl get service/details  deployment.apps/details-v1
NAME              TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
service/details   ClusterIP   10.104.202.24   <none>        9080/TCP   30d

NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/details-v1   1/1     1            1           30d
```

Reply:
agree.